core/src/main/java/net/sourceforge/jnlp/runtime/classloader/DelegatingClassLoader.java

```diff
@@ -0,0 +1,63 @@
+package net.sourceforge.jnlp.runtime.classloader;
+
+import net.adoptopenjdk.icedteaweb.Assert;
+
+import java.net.URL;
+import java.util.Objects;
+
+public class DelegatingClassLoader extends ClassLoader {
+    public static final DelegatingClassLoader instance = new DelegatingClassLoader(Thread.currentThread().getContextClassLoader());
+    private ClassLoader classLoader;
+
+    public static DelegatingClassLoader getInstance() {
+        return instance;
+    }
+
+    private DelegatingClassLoader(ClassLoader loader) {
+        super(loader);
```

Comment:
```suggestion
        super(Assert.requireNonNull(loader, "loader"));
```
More easy way to do it

Reply:
fixed